packages/oak-addon-basic-components/lib/core/Editor/ColorButton.js

```diff
@@ -75,9 +75,23 @@ export default ({ className }) => {
       <DropdownMenu>
         <ColorField
           ref={colorFieldRef}
-          className="oak-color-field py-3"
+          className="py-3"
```

Comment:
`py-3` is a bootstrap class and it should not have been here in the first place 😄  you can remove it